docs/quick-start.md

```diff
@@ -5,7 +5,7 @@ The purpose of this guide is to demonstrate how to easily setup all Asperathos c
 ## 1. Requirements 
 
 1. Have a Kubernetes cluster ready to receive applications;
-2. Have an application image capable consuming items from a ```redis``` workload;
+2. Have an application image capable consuming items from a ```redis``` workload (see [flow of items in redis](job-item-flow.md));
```

Comment:
"capable consuming items" -> "capable of consuming items"

Reply:
Oops, typo! Changed in 59cdc4d104b8a0e7ba3f89aab68eac8ee3d714b7.